### PR TITLE
wallet fix

### PR DESCRIPTION
**Pull request by indulekha-deriv**

### CODE REVIEW

File: cypress/e2e/smoke/Wallets/cryptoWithdrawal.cy.js [hunk 1]
```diff
@@ -24,20 +24,21 @@ function verifyEmailandPerformWithdraw(platform) {
     cy.c_switchWalletsAccount('BTC')
   }
   cy.findByText('Withdraw').should('be.visible').click()
+  cy.log('indu check wallet login' + Cypress.env('walletloginEmail'))
```
Comment (afsaneh-rezaeinejad-deriv) [inline cypress/e2e/smoke/Wallets/cryptoWithdrawal.cy.js:27]: this log is needed?

**Review by abuzar-deriv — APPROVED** [review]
Looks good to me pending @afsaneh-rezaeinejad-deriv's comments.

File: cypress/e2e/smoke/Wallets/addDerivxAccount.cy.js [hunk 2]
```diff
@@ -38,57 +38,45 @@ function verifyTransferFundsMessage(accountType) {
...
-    if (Text.length > 0) {
-      clickAddDerivxButton()
-      verifyDerivxCreation('Real')
-      verifyTransferFundsMessage('Real')
-      expandDemoWallet()
-      clickAddDerivxButton()
-      verifyDerivxCreation('Demo')
-      verifyTransferFundsMessage('Demo')
-    }
+      .should('exist')
```
[the remark this answers is not in the document]
Comment (indulekha-deriv) [reply cypress/e2e/smoke/Wallets/addDerivxAccount.cy.js:53]: Added logic to check if account already created.

File: cypress/e2e/smoke/Wallets/addMt5Account.cy.js [hunk 3]
```diff
@@ -77,33 +74,33 @@ function closeModal() {
...
-    )
-    if (financialText.length > 0) {
-      clickAddMt5Button()
-      verifyJurisdictionSelection('Financial')
-      verifyDerivMT5Creation('Financial')
-      verifyTransferFundsMessage('Financial')
-      closeModal()
-    }
+    cy.findByText('CFDs on derived instruments.')
+      .should('exist')
```
[the remark this answers is not in the document]
Comment (indulekha-deriv) [reply cypress/e2e/smoke/Wallets/addMt5Account.cy.js:86]: Added logic to check if account already created.

File: cypress/e2e/smoke/Wallets/addMt5Account.cy.js [hunk 4]
```diff
@@ -77,33 +74,33 @@ function closeModal() {
...
+      .then(() => {
+        clickAddMt5Button(mt5SVG)
```
[the remark this answers is not in the document]
Comment (indulekha-deriv) [reply cypress/e2e/smoke/Wallets/addMt5Account.cy.js:88]: Updated to check for mt5 account type instead of text content

**Review by meeyun-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/smoke/Wallets/addDerivxAccount.cy.js [hunk 5]
```diff
@@ -38,57 +38,65 @@ function verifyTransferFundsMessage(accountType) {
...
 
 function expandDemoWallet() {
-  cy.get('.wallets-dropdown__button').click()
-  cy.get('.wallets-list-card-dropdown__item-content')
-    .contains('USD Demo Wallet')
-    .click()
+  cy.get('label').find('span').click()
   cy.findByText('USD Demo Wallet').should('be.visible')
 }
+function existingAccountCheck(walletBanner) {
```
Comment (afsaneh-rezaeinejad-deriv) [inline cypress/e2e/smoke/Wallets/addDerivxAccount.cy.js:44]: why walletBanner should be passed as a parameter?

**Review by afsaneh-rezaeinejad-deriv — APPROVED** [review]

**Pull request merged**